Translated docs/adding-comments.md (#91)
gatsbyjs/gatsby-es: docs/docs/adding-comments.md

- [TalkYard](https://www.talkyard.io)
- [Gitalk](https://gitalk.github.io)

Puede también [crear su propio sistema de comentarios](/blog/2019-08-27-roll-your-own-comment-system/), como Tania Rascia escribió en el blog de Gatsby. 
## Usando Disqus para comentarios

En esta guía le mostraremos como implementar Disqus en su blog, ya que tiene características atractivas.

- Es de bajo mantenimiento, es decir, [moderar sus comentarios y mantener su audiencia](https://help.disqus.com/moderation/moderating-101) es fácil.
- Provee [soporte oficial de React](https://github.com/disqus/disqus-react).
- Ofrece un [plan gratis muy generoso](https://disqus.com/pricing).
- [Parece ser por mucho el servicio más usado](https://www.datanyze.com/market-share/comment-systems/disqus-market-share).
- Es fácil comentar: Disqus tiene una gran base de usuarios y la experiencia de integración de nuevos usuarios es rápida. Usted puede registrarse con su cuenta de Google, Facebook o Twitter y los usuarios pueden compartir fácilmente los comentarios que escriben a través de esos canales.

> - La interfaz de Disqus tiene una inconfundible pero discreta apariencia que muchos usuarios reconocerán y confiarán en ella. - Todos los componentes de Disqus son lazy-loaded, es decir, no impactarán en el tiempo de carga de sus posts.

Reviewer: ```suggestion
- La interfaz de Disqus tiene una inconfundible pero discreta apariencia que muchos usuarios reconocerán y confiarán en ella. - Todos los componentes de Disqus son "lazy-loaded", es decir, no impactarán en el tiempo de carga de sus posts.
```

> Sin embargo, tenga en mente que usar Disqus también involucra un sacrificio. Su sitio ya no es completamente estático, pues depende de una plataforma externa para enviar sus comentarios a través de `iframe`s integrados al momento en que el sitio cargue. Además, usted debe considerar las implicaciones de privacidad por permitir que un tercero guarde los comentarios de sus visitantes y potencialmente pueda seguir sus historiales de navegación. Usted puede consultar el [aviso de privacidad de Disqus](https://help.disqus.com/terms-and-policies/disqus-privacy-policy), [preguntas frecuentes de privacidad](https://help.disqus.com/terms-and-policies/privacy-faq) (especialmente la última pregunta acerca del cumplimiento de GDPR) e informar a sus usuarios [como editar su configuración de difusión de datos](https://help.disqus.com/terms-and-policies/how-to-edit-your-data-sharing-settings).

Reviewer: ```suggestion
Sin embargo, ten en mente que usar Disqus también involucra un sacrificio. Tu sitio ya no es completamente estático, pues depende de una plataforma externa para enviar sus comentarios a través de `iframe`s integrados al momento en que el sitio cargue. Además, debes considerar las implicaciones de privacidad por permitir que un tercero guarde los comentarios de tus visitantes y potencialmente puedan seguir sus historiales de navegación. Puedes consultar el [aviso de privacidad de Disqus](https://help.disqus.com/terms-and-policies/disqus-privacy-policy), [preguntas frecuentes de privacidad](https://help.disqus.com/terms-and-policies/privacy-faq) (especialmente la última pregunta acerca del cumplimiento de GDPR) e informar a tus usuarios [como editar su configuración de difusión de datos](https://help.disqus.com/terms-and-policies/how-to-edit-your-data-sharing-settings).
```

> Si estos problemas son más grandes que los beneficios de usar Disqus, usted puede optar por alguna de las siguientes opciones. Aceptamos pull requests para expandir esta guía con instrucciones de configuración para otros servicios.

Reviewer: ```suggestion
Si estos problemas son más grandes que los beneficios de usar Disqus, puedes optar por alguna de las siguientes opciones. Aceptamos pull requests para expandir esta guía con instrucciones de configuración para otros servicios.
```

## Implementando Disqus

![Logo de Disqus](images/disqus-logo.svg)

> Pasos para agregar comentarios de Disqus a su propio blog:

Reviewer: ```suggestion
Pasos para agregar comentarios de Disqus a tu propio blog:
```

> 1. [Registrarse a Disqus](https://disqus.com/profile/signup). Durante el proceso usted tendrá que escoger un nombre corto para su sitio. De esta forma Disqus identificará comentarios viniendo de su sitio. Guarde el nombre para después.

Reviewer: ```suggestion
1. [Registrarse a Disqus](https://disqus.com/profile/signup). Durante el proceso tendrás que escoger un nombre corto para tu sitio. De esta forma Disqus identificará comentarios viniendo de su sitio. Guarda el nombre para después.
```

2. Instalar el paquete Disqus React

```shell
npm install disqus-react
```

> 3. Agregar el nombre que utilizó en el paso 1 como, por ejemplo, `GATSBY_DISQUS_NAME` a sus archivos `.env` y `.env.example` para que las personas que hagan fork a su repositorio sepan que ellos necesitan ingresar este valor para hacer funcionar los comentarios (Usted necesita el prefix de la variable de entorno con `GATSBY_` para que pueda estar [disponible en el código cliente](https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript).)

Reviewer: ```suggestion
3. Agregar el nombre que utilizaste en el paso 1 como, por ejemplo, `GATSBY_DISQUS_NAME` a sus archivos `.env` y `.env.example` para que las personas que hagan fork de tu repositorio sepan que ellos necesitan ingresar este valor para hacer funcionar los comentarios (Necesitas el prefix de la variable de entorno con `GATSBY_` para que pueda estar [disponible en el código cliente](https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript).)
```

```title=.env.example
# habilita comentarios de Disqus para posts en un blog
GATSBY_DISQUS_NAME=insertValue
```

```title=.env
GATSBY_DISQUS_NAME=yourSiteShortname
```

> 4. En el template de su blog post (usualmente `src/templates/post.js`) importe el componente `DiscussionEmbed`.

Reviewer: ```suggestion
4. En el template de tu blog post (usualmente `src/templates/post.js`) importa el componente `DiscussionEmbed`.
```

import { DiscussionEmbed } from "disqus-react"
```

> Luego defina su objeto de configuración de Disqus

Reviewer: ```suggestion
Luego define tu objeto de configuración de Disqus
```

}
```

> Donde `identifier` debe ser un string o número que identifique exclusivamente a ese post. Puede ser el slug del post, título o un ID. Finalmente, agregue `DiscussionEmbed` al JSX del template de su post.

Reviewer: ```suggestion
Donde `identifier` debe ser un string o número que identifique exclusivamente a ese post. Puede ser el slug del post, título o un ID. Finalmente, agrega `DiscussionEmbed` al JSX del template de tu post. 
```

  </Global>
)
```

> Y eso es todo. Ahora debe poder ver la forma de comentarios de Disqus debajo del post de su blog [de esta forma](https://janosh.io/blog/disqus-comments#disqus_thread). ¡Feliz blogging!

Reviewer: ```suggestion
Y eso es todo. Ahora debes poder ver el formulario de comentarios de Disqus debajo del post de tu blog [de esta forma](https://janosh.io/blog/disqus-comments#disqus_thread). ¡Feliz blogging!
```
